ruby/ql/lib/codeql/ruby/frameworks/StandardLibrary.qll

```diff
@@ -335,3 +336,18 @@ class ModuleEvalCallCodeExecution extends CodeExecution::Range, DataFlow::CallNo
 
   override DataFlow::Node getCode() { result = this.getArgument(0) }
 }
+
+/** Flow summary for `Regexp.escape`. */
+class RegexpEscapeSummary extends SummarizedCallable {
+  RegexpEscapeSummary() { this = "Regexp.escape" }
+
+  override MethodCall getACall() {
+    result = API::getTopLevelMember("Regexp").getAMethodCall("escape").asExpr().getExpr()
```

Comment:
[`Regexp.quote`](https://ruby-doc.org/core-3.0.2/Regexp.html#method-c-quote) is an alias for `Regexp.escape`, so we could add that here too.